packages/react-rest/vite.config.ts

```diff
@@ -1,3 +1,4 @@
+/// <reference types="vitest" />
```

Comment:
sometimes I needed this, sometimes I didn't. I couldn't figure out the pattern

Reply:
i've had to do it whenever i add the `test` property in `defineConfig`